ui/src/views/application/component/AccessSettingDrawer.vue

```diff
@@ -222,10 +222,13 @@ const submit = async () => {
   formRef.value?.validate(async (valid) => {
     if (valid) {
       try {
-        await applicationApi.updatePlatformConfig(id, configType.value, form[configType.value])
-        MsgSuccess('配置保存成功')
-        closeDrawer()
-        emit('refresh')
+        applicationApi
+          .updatePlatformConfig(id, configType.value, form[configType.value], loading)
+          .then(() => {
+            MsgSuccess('配置保存成功')
+            closeDrawer()
+            emit('refresh')
+          })
       } catch {
         MsgError('保存失败，请检查输入或稍后再试')
       }
```

Comment:
The provided code looks generally correct. However, there is an improvement opportunity to manage the loading state more explicitly:

1. Instead of using global `loading` variable, pass it as a parameter to the API call. This makes testing easier and keeps components decoupled from shared state.

```javascript
applicationApi.updatePlatformConfig(
  id,
  configType.value,
  form[configType.value],
  loading // Passing the loading flag
).then(() => {
  MsgSuccess('配置保存成功')
  closeDrawer()
  emit('refresh')
}).catch((error) => {
  msgError(`保存失败，错误信息: ${error.message}`); // Handling errors separately
});
```

This change enhances reusability and maintainability by isolating the loading state within each component.